tests/test_api/test_builds.py

```diff
@@ -83,3 +84,150 @@ async def test_build_create_without_permissions(
         )
         assert response.status_code == self.status_codes.HTTP_403_FORBIDDEN
         self.headers["Authorization"] = old_token
+
+
+class TestModularBuilds(BaseAsyncTestCase):
+    async def test_multilib_virt(
+        self,
+        get_multilib_packages_from_pulp,
+        enable_beholder,
+        mock_beholder_call,
+        multilib_virt_with_artifacts: str,
+        modules_artifacts: dict,
+        virt_build_payload: dict,
+        virt_modular_build: Build,
+        start_modular_virt_build,
+        virt_build_done,
+        tmp_path,
```

Comment:
you can move common fixtures such as `get_multilib_packages_from_pulp`, `enable_beholder` and etc. in `usefixtures` decorator
```python
@pytest.mark.usefixtures(
    "get_multilib_packages_from_pulp",
    "enable_beholder",
    ...
)
class TestModularBuilds(BaseAsyncTestCase):
    ...
```